lock-api/src/main/java/com/palantir/lock/client/LockLeaseService.java

```diff
@@ -40,19 +42,31 @@
 import com.palantir.logsafe.Preconditions;
 
 class LockLeaseService {
+    private static final boolean HAVE_ROLLED_OUT_CONJURE_CHANGES_INTERNALLY = false;
     private final NamespacedTimelockRpcClient delegate;
+    private final NamespacedConjureTimelockService conjureDelegate;
     private final UUID clientId;
     private final CoalescingSupplier<LeaderTime> time;
 
     @VisibleForTesting
-    LockLeaseService(NamespacedTimelockRpcClient timelockRpcClient, UUID clientId) {
+    LockLeaseService(
+            NamespacedTimelockRpcClient timelockRpcClient,
+            NamespacedConjureTimelockService conjureDelegate,
+            UUID clientId) {
         this.delegate = timelockRpcClient;
+        this.conjureDelegate = conjureDelegate;
         this.clientId = clientId;
-        this.time = new CoalescingSupplier<>(timelockRpcClient::getLeaderTime);
+        if (HAVE_ROLLED_OUT_CONJURE_CHANGES_INTERNALLY) {
```

Comment:
right, exactly - basically set to true and inline

Reply:
do we want to add the ability to fallback in the case things are borked? or actually just have this be a feature flag until we're happy with it? since this is inside the client and will be annoying to coordinate in the presence of issues.

Reply:
@felixdesouza Don't think there's very much code (beyond generated stuff) on the client side? The server impl is different and maybe we can have a flag there, but it's a lot easier to blacklist stuff.